src/components/Group.js

```diff
@@ -130,8 +139,22 @@ export default function Group(props) {
   </div>;
 }
 
-function DataMarker() {
-  return (
-    <div title="Section contains data" class="bio-properties-panel-dot"></div>
-  );
+function DataMarker(props) {
+  const {
+    edited,
+    hasErrors
+  } = props;
+
+  if (hasErrors) {
+    return (
+      <div title="Section contains an error" class="bio-properties-panel-dot error"></div>
```

Comment:
Consequence of https://github.com/bpmn-io/properties-panel/pull/256/files#r1265083126

```suggestion
      <div title="Section contains an error" class="bio-properties-panel-dot bio-properties-panel-dot--error"></div>
```